sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuGetJsonObject.scala

```diff
@@ -143,17 +144,19 @@ class GpuGetJsonObjectMeta(
     conf: RapidsConf,
     parent: Option[RapidsMeta[_, _, _]],
     rule: DataFromReplacementRule
-  ) extends BinaryExprMeta[GetJsonObject](expr, conf, parent, rule) {
+  ) extends BinaryExprMeta[GetJsonObject](expr, conf, parent, rule) with Logging {
 
   override def tagExprForGpu(): Unit = {
     val lit = GpuOverrides.extractLit(expr.right)
     lit.foreach { l =>
       val instructions = JsonPathParser.parse(l.value.asInstanceOf[UTF8String].toString)
       if (!conf.isLegacyGetJsonObjectEnabled) {
+        logInfo("get_json_object executes through new code path.")
```

Comment:
logInfo is typically suppressed on the driver

Don't we already see if the "new code path"  is taken based on whether SparkPlan exhibits GpuGetJsonObject vs GpuGetJsonObjectLegacy

Reply:
I have no idea. @viadea ?

Reply:
I think as long as the query plan can show legacy vs latest from Spark UI, then we should be good.
I do not think we need driver log extra message.